impl/src/rendering/templates/crate.BUILD.template

```diff
@@ -30,7 +30,7 @@ licenses([
 
 # Generated Targets
 {%- set crate_name_sanitized = crate.pkg_name | replace(from="-", to="_") -%}
-{%- if crate.build_script_target -%}
+{%- if crate.build_script_target %}
 {%      include "templates/partials/build_script.template" %}
```

Comment:
```suggestion

{%      include "templates/partials/build_script.template" %}
```

Reply:
(Just the empty line, I cannot seem to find a way to add new lines without removing anything)

Reply:
Thanks!